import-issues.py

```diff
@@ -341,7 +361,14 @@ def main():
     for card_data in trello_data['cards']:
         cards_log.debug("Card %s", card_data['name'])
         card = Card(card_data, args, labels_mapper)
-        ok = card.save()
+
+        list_name = [list['name'] for list in trello_data['lists'] if list['id'] == card_data["idList"]].pop()
```

Comment:
i think this would be better as a generator with next so that there's not a whole array allocated:
```python
list_name = next((
    list['name']
    for list in trello_data['lists']
    if list['id'] == card_data["idList"]
))
```

what do you think?
(note: code untested)

also, if the line could be split across multiple lines (like in the example), that'd be great. the full line above makes it difficult to skim